app/api/helpers/db.py

```diff
@@ -115,3 +116,23 @@ def get_count(query):
     count_q = query.statement.with_only_columns([func.count()]).order_by(None)
     count = query.session.execute(count_q).scalar()
     return count
+
+
+def get_new_slug(name, model):
```

Comment:
Great job, but this should take `(model, name)`, so it can be called like `get_new_slug(EventLocation, name)`

Reply:
Okay.I will do it.